public/static/js/build.js

```diff
@@ -247,13 +248,10 @@ mciModule.controller('BuildViewController', function($scope, $http, $timeout, $r
         return new Date(x.Task.start_time) > new Date(0) && new Date(x.Task.finish_time) > new Date(0)
       }
     )
-    $scope.totalTimeMS = _.reduce(
-      _.map(finishedOnly,
-        function(x){return new Date(x.Task.finish_time) - new Date(x.Task.start_time)}),
-        function(sum, el){return sum+el},
-        0)
-    };
 
+    $scope.totalTimeMS = _.reduce(_.pluck(finishedOnly, "Task"), function(x, y){return x.time_taken+y.time_taken}, 0) / nsPerMs;
```

Comment:
Where they are display tasks, this should get the display tasks xor their execution tasks. How does this handle that?

Reply:
This is no longer relevant as the calculation of totalTime and makespan was moved to the backend.
But the answer _was_ (:grin:) the only tasks included in the count were those in the build's task cache, which excluded exec tasks.